### ISSUE TITLE
Porting of blizz_vanisher

### ISSUE THREAD

**Issue by Graham-72** [backdrop-ops/contrib#295]
Has anyone else had a look at this Drupal module?

I am starting to look at it because of GDPR requirements and if no-one else is working on it I shall endeavour to port it to Backdrop.

**Comment by Graham-72** [comment]
I am making progress on the port but it is not yet ready for release. The D7 version uses class EntityAPIControllerExportable which is not included in Backdrop, so I am trying the new entity_plus module, with some success.

**Comment by offthehookdesign** [comment]
Hi Graham, I've posted in the EU Cookies issues queue, not realising you've posted here. I started to port Blizz Vanisher myself, and got the same EntityAPIControllerExportable class problem. I've not enough skills to do much more with it so it's exciting to see you're on the case. I do have a pet peeve with this module, and that's the name! 

How do you feel about changing it, perhaps to something like GDPR Cookies, which is a bit more descriptive of the modules function?

**Comment by olafgrabienski** [comment]
I agree, changing the name is a good idea. "blizz" is an 'artificial' term used for a couple of quite different modules, all created by one agency (comm-press), see https://comm-press.de/blizz-for-thunder-cms (in German language).

**Comment by Graham-72** [comment]
@olafgrabienski thanks for the information. I will change to the new name.

**Comment by sbgraphicdesign** [comment]
@Graham-72 very many thanks for your effort with this. Am very excited for this to be available. 

Would you like me to communicate with the entity plus devs to ask when a release date would be likely?

Also I do have some ideas for enhancing this module. Would you prefer for me to wait until you have a repository in order to raise these as issues? 

I wish I could help more but my coding skills are suspect, but happy to assist with any UI, styling or documentation you may need.

**Comment by Graham-72** [comment]
I have now transferred my port of blizz_vanisher to backdrop-contrib. It is available at https://github.com/backdrop-contrib/blizz_vanisher. Any issues can now be raised there.

It requires the installation of module entity_plus which is available from https://github.com/backdrop-contrib/entity_plus.

I found problems with the tarteaucitron scripts included in the D7 release (1.3.0) and I have replaced thos with a more recent version.

My next step is to release this module with a new name, GDPR Cookies, for installation in Backdrop.

@sbgraphicdesign have you experience of using this in D7? It wouldn't work for me, probably because of the old version of the scripts. Also, I find it a bit tricky to configure and will raise an issue about this.

**Comment by sbgraphicdesign** [comment]
Hi @Graham-72 , many thanks for all the work you've done so far to get this ported. I've now replied to your issue on the module page, which I hope will prove helpful.

I can certainly assist with some documentation and perhaps advice for streamlining the UI a touch once we've ironed out these teething issues.

I think this is a really important module (certainly for those of us in the EU), and hopefully we can get some other developers on board.
